components/Posts/DescribePost.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="fr-p-3w bg-white">
+    <RequiredExplanation />
+    <fieldset
+      class="fr-fieldset min-width-0"
+      aria-labelledby="description-legend"
+    >
+      <legend
+        id="description-legend"
+        class="fr-fieldset__legend"
+      >
+        <h2 class="subtitle subtitle--uppercase !mb-3">
+          {{ $t("Description") }}
+        </h2>
+      </legend>
+      <div class="fr-fieldset__element">
+        <InputGroup
+          v-model="form.name"
+          :label="$t('Post title')"
+          :required="true"
+          :has-error="!!getFirstError('name')"
+          :has-warning="!!getFirstWarning('name')"
+          :error-text="getFirstError('name')"
+          @blur="touch('name')"
+        />
+      </div>
+      <div class="fr-fieldset__element">
+        <InputGroup
+          v-model="form.headline"
+          :label="$t('Headline')"
+          :has-error="!!getFirstError('headline')"
+          :has-warning="!!getFirstWarning('headline')"
+          :error-text="getFirstError('headline')"
+          required
+          @blur="touch('headline')"
+        />
+      </div>
+      <div class="fr-fieldset__element">
+        <RadioButtons
+          v-model="form.body_type"
+          :label="t('Content type')"
+          class="!mb-0"
+          :options="[
+            { value: 'html', label: t('HTML') },
+            { value: 'markdown', label: t('Markdown') },
+          ]"
+        />
+      </div>
+      <div class="fr-fieldset__element">
+        <TagsSelect
+          v-model="form.tags"
+          :error-text="getFirstError('tags')"
+          :warning-text="getFirstWarning('tags')"
+        />
+      </div>
+      <div class="fr-fieldset__element">
+        <UploadGroup
+          :label="$t('Cover')"
+          type="drop"
+          accept="jpg,jpeg,png"
+          :multiple="true"
+          :hint-text="$t('Max size: 4 Mb. Accepted formats: JPEG, JPG and PNG.')"
+          show-label
+          required
+          @change="setFiles"
+        />
+        <div
+          v-if="imagePreview"
+          class="text-align-center"
+        >
+          <NuxtImg
+            :src="imagePreview"
+            class="border mx-auto max-h-40 aspect-square"
+            loading="lazy"
+            alt=""
+          />
+        </div>
+      </div>
+    </fieldset>
+    <div
+      class="fr-grid-row"
+      :class="{ 'fr-grid-row--right': type === 'update', 'justify-between': type === 'create' }"
+    >
+      <BrandedButton
+        v-if="type === 'create'"
+        color="secondary"
+        @click="$emit('previous')"
+      >
+        {{ $t('Previous') }}
+      </BrandedButton>
+      <BrandedButton
```

Comment:
This one is missing a loader

Reply:
I don't see a loader in `DescribeDataset` too… And this component is not the one doing the update so I may need to think of the best solution to do this… Could we postpone?